File: core/src/main/java/com/example/coinstest/domain/CurrencyInRub.java
package com.example.coinstest.domain;

public final class CurrencyInRub {
    public final String name;
    public final String date;
    public final float value;

    public CurrencyInRub(String name, String date, float value) {
        this.name = name;
        this.date = date;
        this.value = value;
    }
}

File: domain/src/main/java/com/andysklyarov/domain/model/CurrencyInRub.kt
package com.andysklyarov.domain.model

import org.threeten.bp.LocalDate


data class CurrencyInRub(
    val fullName: String,
    val chCode: String,
    val date: LocalDate,
    val nom: Int,
    val value: Float,
    val diff: Float
)